### PR TITLE
Remove backfill historical data on compressed chunks

### LINKED ISSUE #3559: [Docs RFC] Remove Backfill historical data on compressed chunks
- Remove https://docs.timescale.com/use-timescale/latest/compression/backfill-historical-data/#backfill-historical-data-on-compressed-chunks (+ fix links, add a redirect)
- Add a section on disabling a policy to https://docs.timescale.com/use-timescale/latest/compression/compression-policy/

### PR DESCRIPTION
**Pull request by atovpeko**

### CODE REVIEW

**Review by billy-the-fish — APPROVED** [review]
LGTM.

File: use-timescale/compression/compression-policy.md [hunk 1]
```diff
@@ -59,6 +59,20 @@ SELECT * FROM timescaledb_information.jobs
 
 For more information, see the API reference for [`timescaledb_information.jobs`][timescaledb_information-jobs].
 
+## Pause compression policy
+
+To disable a compression policy temporarily:
```
Comment (jamessewell) [inline use-timescale/compression/compression-policy.md]: We should probably tell people how to find the jobid here.

SELECT * FROM timescaledb_information.jobs where proc_name = 'policy_compression' AND relname = '<hypertablename>'

File: use-timescale/compression/decompress-chunks.md [hunk 2]
```diff
@@ -70,7 +70,6 @@ SELECT tableoid::regclass FROM metrics
  _timescaledb_internal._hyper_72_37_chunk
 ```
 
-[backfill]: /use-timescale/:currentVersion:/compression/backfill-historical-data/
 [api-reference-decompress]: /api/:currentVersion:/compression/decompress_chunk/
 [api-reference-alter-job]: /api/:currentVersion:/actions/alter_job/
-
+[backfilling_script]: https://github.com/timescale/timescaledb-extras/blob/master/backfill.sql
```
Comment (jamessewell) [inline use-timescale/compression/decompress-chunks.md]: I don't think we need to link this script at all, it's not relevant anymore.

File: use-timescale/compression/compression-policy.md [hunk 3]
```diff
@@ -61,7 +61,11 @@ For more information, see the API reference for [`timescaledb_information.jobs`]
 
 ## Pause compression policy
 
-To disable a compression policy temporarily:
+To disable a compression policy temporarily, find the corresponding job ID and then call `alter_job` to pause it:
+
+```sql
+SELECT * FROM timescaledb_information.jobs where proc_name = 'policy_compression' AND relname = ''
+```
```
Comment (jamessewell) [inline use-timescale/compression/compression-policy.md:68]: Sorry I wasn't clear, relname would be set to the hypertable name. So however we show that in other places in the docs `<hypertablename>` maybe?

Comment (atovpeko) [reply use-timescale/compression/compression-policy.md:68]: Oooh, missed that, sorry. This page seems to be using both - placeholders and examples - so I aligned all the commands to refer to the same hypertable called "example". Hope this works.

**Review by jamessewell — APPROVED** [review]

**Comment by jamessewell** [comment]
LGTM!

**Pull request merged**